x/bank/types/params_legacy.go

```diff
@@ -18,6 +23,37 @@ func ParamKeyTable() paramtypes.KeyTable {
 // Deprecated: ParamSetPairs implements params.ParamSet
 func (p *Params) ParamSetPairs() paramtypes.ParamSetPairs {
 	return paramtypes.ParamSetPairs{
+		paramtypes.NewParamSetPair(KeySendEnabled, &p.SendEnabled, validateSendEnabledParams),
```

Comment:
can remove this now?

Reply:
still need, check in [here](https://github.com/crypto-org-chain/cosmos-sdk/blob/dcbb61e9b5f99dce3de93cb1b0e54523f8f34c00/x/params/types/subspace.go#L168)

Reply:
I see, maybe `GetRaw`?

another thing is, if we register the pair here, we need to explicitly clear the field when setting the new parameters, because it's not useful anymore.